### PR TITLE
Sqlserver delay config

### LINKED ISSUE #1835: Create documentation for the new queue peeker delay configuration api
This is related to Particular/NServiceBus.SqlServer#271

### PR DESCRIPTION
**Pull request by HEskandari**
Waiting for the api to get updated and published to myget before I can make this build green. Resolves #1835

### CODE REVIEW

**Comment by HEskandari** [comment]
@Particular/docs-maintainers This is ready for review.

**Comment by SimonCropp** [comment]
do u want it in the menu?

File: nservicebus/sqlserver/queuepeek-settings.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+reviewed: 2016-09-01
+component: SqlServer
+versions: '[3.0,)'
+tags:
+ - SQL Server
+---
+
+The SQL Server transport uses database tables as conceptual queues to read messages from. Current implementation uses a pull mechanism to 'Peek' if there are any messages waiting to get processed. There is a delay between each peek which defaults to one second. Depending on the system, the amount of messages and the load, the peek delay time may be tweaked. 
+
+The acceptable range for this setting is between 100 milliseconds to 10 seconds. If a value higher than the maximum recommended settings is used, a warning message will be logged.
```
Comment (SimonCropp) [inline nservicebus/sqlserver/queuepeek-settings.md]: since we only warn on outside the range and nor error i recommend "acceptable" => "recommended"

Comment (SimonCropp) [inline nservicebus/sqlserver/queuepeek-settings.md]: can we add a statement about why this is the recommended range?

**Pull request merged**